src/main/groovy/net/anshulverma/gradle/estilo/checkstyle/checks/ConfigFileLoader.groovy

```diff
@@ -31,12 +33,16 @@ class ConfigFileLoader {
   CheckType checkType
 
   CheckstyleConfig load() {
-    loadResource("/${checkType.filename}")
+    if(checkType == CheckType.CUSTOM){
+      loadUrl(new File(checkType.filename).toURI().toURL())
+    }else{
+      loadUrl(this.getClass().getResource("/${checkType.filename}"))
+    }
   }
 
-  private CheckstyleConfig loadResource(String resource) {
-    File tmpFile = File.createTempFile('checkstyle', checkType.filename)
-    FileUtils.copyURLToFile(this.getClass().getResource(resource), tmpFile)
+  private CheckstyleConfig loadUrl(URL url){
+    File tmpFile = File.createTempFile('checkstyle', Paths.get(checkType.filename).getFileName().toString())
```

Comment:
I had to use `Paths` because filenames for custom files that contained a slash produced error with the temp file creation. So I use `Paths` to get the actual filename of the file.